data-access/mandataris.ts

```diff
@@ -5,14 +5,90 @@ import {
   sparqlEscapeDateTime,
   sparqlEscapeUri,
 } from 'mu';
-import { CSVRow, CsvUploadState, MandateHit, Term } from '../types';
+import {
+  CSVRow,
+  CsvUploadState,
+  MandateHit,
+  Term,
+  TermProperty,
+} from '../types';
 import moment from 'moment';
 import { v4 as uuidv4 } from 'uuid';
 import { MANDATARIS_STATUS, PUBLICATION_STATUS } from '../util/constants';
 import { sparqlEscapeTermValue } from '../util/sparql-escape';
-import { findFirstSparqlResult } from '../util/sparql-result';
+import {
+  findFirstSparqlResult,
+  getBooleanSparqlResult,
+} from '../util/sparql-result';
 import { TERM_MANDATARIS_TYPE } from './mandatees-decisions';
 
+export const mandataris = {
+  isValidId,
+  findCurrentFractieForPerson,
+  getPersonWithBestuursperiode,
+};
+
+async function isValidId(id: string): Promise<boolean> {
+  const askQuery = `
+    PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+    PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+
+    ASK {
+      ?mandataris a mandaat:Mandataris;
+        mu:uuid ${sparqlEscapeString(id)}.
+    }
+  `;
+  const sparqlResult = await querySudo(askQuery);
```

Comment:
If you remove this sudo, you can use this query as authorization check. If you don't find the id, you either don't have access to it, or it doesn't exist.